plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObject.class.php

```diff
@@ -69,12 +71,17 @@ public function recursivelyAddInformationObjects($parentId, $totalRows, $options
 
       try
       {
+        // Index document if within paging limits
         $node = new arElasticSearchInformationObjectPdo($item->id, $options);
-        $data = $node->serialize();
 
-        QubitSearch::getInstance()->addDocument($data, 'QubitInformationObject');
+        if (!is_numeric($skip) || (self::$counter > $skip && self::$counter <= ($skip + $totalRows)))
```

Comment:
If `$skip` defaults to zero, then you don't need the `!is_numeric($skip)` check.